robottelo/hosts.py

```diff
@@ -1546,6 +1546,19 @@ def set_rex_script_mode_provider(self, mode='ssh'):
         if result.status != 0:
             raise SatelliteHostError(f'Failed to enable pull provider: {result.stdout}')
 
+    def set_installer_arg(self, *args):
```

Comment:
```suggestion
    def set_installer_arg(self, *args, timeout='20m'):
```
Does it make sense to pass the timeout as argument (and use lower) since some installer runs may be faster or slower?